launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -26,21 +26,6 @@
 
   <let name="vehicle_model_pkg" value="$(find-pkg-share $(var vehicle_model)_description)"/>
 
-  <group if="$(var scenario_simulation)">
-    <include file="$(find-pkg-share fault_injection)/launch/fault_injection.launch.xml">
-      <arg name="config_file" value="$(var fault_injection_param_path)"/>
-    </include>
-  </group>
-
-  <!-- Dummy Perception -->
-  <group if="$(var launch_dummy_perception)">
```

Comment:
Could you bring back condition too? Otherwise this PR will make this launch file act differently than before.